pages/agent/gcloud.md.erb

```diff
@@ -0,0 +1,183 @@
+# Running Buildkite Agent on Google Cloud Platform
+
+The Buildkite Agent can be run on [Google Cloud Platform](https://cloud.google.com). You may like to run the agent using individual VM instances on Google Compute Engine for fine control over long–lived agents. Or you may like to run a scalable cluster of agents using Docker and Kubernetes on the Google Container Engine to perform Docker–based builds.
+
+<%= toc %>
+
+## Running the agent on Google Compute Engine
+
+To run the agent on your own [Google Compute Engine](https://cloud.google.com/compute) instance use whichever installer matches your instance type. For example, launch an instance using the [latest Ubuntu LTS image](https://console.cloud.google.com/launcher?cat=OS&q=ubuntu) and follow [our Ubuntu setup instructions](/docs/agent/ubuntu).
+
+## Running the agent on Google Container Engine
+
+[Google Container Engine](https://cloud.google.com/container-engine) can also run the agent as a [Docker](https://www.docker.com) container using [Kubernetes](https://kubernetes.io). Make sure to run the container in [privileged mode](https://kubernetes.io/docs/user-guide/pods/#privileged-mode-for-pod-containers) and map the Docker socket and binary into the container as volumes if you wish to [run Docker–based builds](http://buildkite.dev/docs/builds/docker-containerized-builds). For example:
+
+Start a Google Container Engine cluster [through the console](https://console.cloud.google.com/kubernetes/add):
+
+<%= image "create-a-cluster.png", size: "2879x1277", alt: "Screenshot of creating a Google Container Engine cluster via the Google Cloud Console" %>
+
+Open [Google Cloud Shell](https://cloud.google.com/shell/) or your own console with [gcloud](https://cloud.google.com/sdk/gcloud/) installed and authenticated. You'll need to configure kubectl to talk to your new cluster. The console includes a "Connect" button which shows the exact command to run:
+
+<%= image "connect-to-cluster.png", size: "2878x1531", alt: "Screenshot of cluster connection settings in the Google Cloud Console" %>
+
+```
+$ gcloud container clusters get-credentials INSERT-YOUR-CLUSTER-NAME \
+    --zone INSERT-YOUR-ZONE --project INSERT-YOUR-PROJECT
+Fetching cluster endpoint and auth data.
+kubeconfig entry generated for YOUR-CLUSTER-NAME
+```
+
+Double check it's all working:
```

Comment:
Would be good to say how you're checking it's working in this bit, so that the reader doesn't have to parse the code block to figure out what's going on.